src/test/resources/packagerConfig/emptyCascSection.json

```diff
@@ -2,7 +2,7 @@
   "bundle": {
     "artifactId" : "jenkins-all-latest",
     "title" : "Jenkins WAR - all latest",
-    "desc" : "New Jenkins Version"
+    "description" : "New Jenkins Version"
   },
   "war" : {
     "jenkinsVersion": "2.107.3"
```

Comment:
Random: want to push jenkinsVersion to the most recent version: 2.248.

Reply:
@kwhetston It actually does not matter, since we are just testing the format